### PR TITLE
fix(storybook): ensure default `components/` dir exists

### PR DESCRIPTION
**Pull request by benjamincanac**
Ensure `components/` folder exists before pushing default stories path.

## Types of changes

- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

The `~/components/**/*.stories.@(ts|js)` path is forced and an error is thrown when missing. 

I'm using `@nuxtjs/storybook` to create docs for a nuxt module and I don't have a `components/` folder. In my `docs/` folder I just have a `nuxt.config.js`:

```js
import { join } from 'path'
import module from '..'

export default {
  components: true,
  buildModules: [module],
  storybook: {
    stories: [join(__dirname, '../lib/components/**/**.stories.js')]
  }
}
```
 
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

### CODE REVIEW

**Review by Atinux — APPROVED** [review]

**Comment by pi0** [comment]
Can you please explain use case at least in pull-request description?

**Comment by benjamincanac** [comment]
@pi0 The description has been updated.

**Comment by pi0** [comment]
Thanks for details @benjamincanac :heart: 

> The ~/components/**/*.stories.@(ts|js) path is forced and an error is thrown when missing.
Maybe for this bug fix we can check `srcDir/components` existence before adding to defaults?

Also would be really nice if we call a hook like `storybook:options` to allow modules problematically override behavior or push their own story path.

**Comment by farnabaz** [comment]
Thank you @benjamincanac 💯  
I think checking `srcDir/components` existence is a good solution for this bug because it adds less complexity to the source code.

>Also would be really nice if we call a hook like `storybook:options` to allow modules problematically override behavior or push their own story path.

Indeed this is a perfect way to improve integrity between community modules and storybook.

**Comment by benjamincanac** [comment]
@pi0 @farnabaz I updated the PR according to the requested changes.

File: src/index.ts [hunk 1]
```diff
@@ -169,10 +169,13 @@ function nuxtStorybookOptions (options) {
   if (!srcDir.startsWith('/')) {
     srcDir = path.resolve(options.rootDir, srcDir)
   }
-  nuxtStorybookConfig.stories = [
-    '~/components/**/*.stories.@(ts|js)',
-    ...nuxtStorybookConfig.stories
-  ].map(story => upath.normalize(story
+
+  const storiesDir = path.resolve(options.rootDir, 'components/**/*.stories.@(ts|js)')
+  if (fsExtra.existsSync(storiesDir)) {
```
Comment (pi0) [inline src/index.ts:174]: Shouldn't we check existence without **/stories in the path?

Comment (pi0) [reply src/index.ts:174]: PS: ~ alias (default) refers to srcDir not rootDir

Comment (benjamincanac) [reply src/index.ts:174]: @pi0 Indeed, I've updated!

**Review by pi0 — APPROVED** [review]

**Comment by farnabaz** [comment]
Thank you @benjamincanac

**Pull request merged**